core/src/main/resources/schema/h2/h2-create-schema.sql

```diff
@@ -39,3 +39,17 @@ CREATE TABLE IF NOT EXISTS "snapshot" (
     PRIMARY KEY("persistence_id","sequence_number")
     );
 
+CREATE TABLE IF NOT EXISTS "state" (
+    "global_offset" BIGINT NOT NULL AUTO_INCREMENT,
+    "persistence_id" VARCHAR(255) NOT NULL,
+    "sequence_number" BIGINT NOT NULL,
```

Comment:
@octonato Calling it `version` or `revision` sounds good to me. But since @patriknw used this term in the API definitions, I decided to use `sequence_number`. Maybe we should ping @patriknw once for this change ?

Reply:
We can rename it to `revision` and then it should be done in all places Akka DurableStateBehavior and store apis.

> optimistic locking implementation

It can be used for other things also, such as de-duplication in consumers of changes, but that works fine with naming it `revision`.